Debug App/Tests/Unit Tests/Extensions/StringExtensionTests.swift

```diff
@@ -198,7 +198,7 @@ final class StringExtensionTests: XCTestCase {
         XCTAssertThrowsError(try "".validateExpiryDateString())
         XCTAssertThrowsError(try "01/2022".validateExpiryDateString())
         XCTAssertThrowsError(try "08/2022".validateExpiryDateString())
-        XCTAssertNoThrow(try "01/2023".validateExpiryDateString())
+        XCTAssertNoThrow(try almostOneYearAgoDateString().validateExpiryDateString())
```

Comment:
@jnewc not sure if this will step on the toes of another fix - this was my attempt to get around the failing test :)
I can revert if needed

Reply:
No this is great, let's keep it